cypress/e2e/devpool.cy.ts

```diff
@@ -0,0 +1,146 @@
+describe("DevPool", () => {
+  let issue1: Record<string, unknown>;
+  let issue2: Record<string, unknown>;
+  let loginToken: Record<string, unknown>;
+  let githubUser: Record<string, unknown>;
```

Comment:
Try setting all the correct types by importing them


Check this example https://github.com/ubiquity/github-labels-processor/blob/5f3921da1fa944d340cd375c3dfc4503531ebee5/src/github-types.ts

Reply:
Can do. I didn't bother because they are populated from a string, so type checking is not relevant as I don't manipulate them. Will change it